docs/admin/release_notes/version_1.6.md

```diff
@@ -0,0 +1,12 @@
+# v1.6 Release Notes
+
+- Add ability to generate ConfigPlans for configurations that need to be deployed, based on multiple plan types.
+- Add a job that can deploy config_set based on a generated ConfigPlan object.
+- Add functionality to compliance result to provide a Remediation plan.
+- Supports Nautobot >=1.6.1,<2.0.0.
+
+## v1.6.0 - 2023-09
+
+### Added
+
+- [573](https://github.com/nautobot/nautobot-plugin-golden-config/pull/573) - Add ConfigPlan and Remediation features.
```

Comment:
My mistake, should have put it as deploy as well. Can you make this 3 lines pointing to the same PR. It should be something like:

- 573 - Added the ability to generate remediation configurations and store in ConfigRemediation model
- 573 - Added the ability to generate configurations that you plan to deploy from a variety of methods, such as Remediation, intended, manual, etc. via the ConfigPlan model
- 573 - Added the ability to Deploy configurations from the ConfigPlan configurations to your network devices.

Reply:
of course